tests/unittests/OnnxImporterTest.cpp

```diff
@@ -1814,6 +1814,46 @@ TEST(onnx, importCastToInt64) {
   importCast("castToInt64.onnxtxt", "data", {1, 2, 2, 2}, ElemKind::Int64ITy);
 }
 
+TEST(onnx, cast_32_64) {
```

Comment:
Is an ONNX importer test necessary here? I think a GraphOptzTest would be easier to add and upkeep, and more directly tests this PR. Also, this test does not test the float->int path, nor verify int->float works fine. Please add more coverage.